frontend/packages/otr/src/redux/reducers/clerkInterpreterOverview.ts

```diff
@@ -0,0 +1,55 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { APIResponseStatus } from 'shared/enums';
+
+import {
+  ClerkInterpreter,
+  ClerkInterpreterOverviewState,
+} from 'interfaces/clerkInterpreter';
+
+const initialState: ClerkInterpreterOverviewState = {
+  interpreter: undefined,
+  overviewStatus: APIResponseStatus.NotStarted,
+  interpreterDetailsStatus: APIResponseStatus.NotStarted,
+  qualificationDetailsStatus: APIResponseStatus.NotStarted,
```

Comment:
Nimeäminen: tässä `interpreterDetailsStatus` ja `qualificationDetailsStatus` viittaavat vissiin tietojen päivittämiseen palvelimella (POST/PUT), kun taas `overviewStatus` taitaa olla tietojen hakemista palvelimelta (GET)? Jos näin on, niin se voisi heijastua kenttien nimissä.

Reply:
Vaikuttaa ihan OK:lta mun mielestä tämä.